core/src/main/scala/org/apache/spark/memory/StaticMemoryManager.scala

```diff
@@ -104,6 +104,8 @@ private[spark] class StaticMemoryManager(
 
 private[spark] object StaticMemoryManager {
 
+  private val MIN_MEMORY_BYTES = 450 * 1024 * 1024
```

Comment:
How about a trait containing the **RESERVED_SYSTEM_MEMORY_BYTES** in the https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/memory/MemoryManager.scala file? 
Both the **StaticMemoryManager** and the **UnifiedMemoryManager** objects could extend this trait and reference the constant.